Assets/Scripts/HoldObject.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class HoldObject : MonoBehaviour
+{
+    private GameObject pickedUpObject;
+    public float interactionDistance = 10f;
+
+    void Update()
+    {
+        if (Input.GetKeyDown("e") && pickedUpObject == null)
+        {
+            PickUp();
+        }
+
+        if (pickedUpObject != null)
+        {
+            CarryObject();
+
+            if (Input.GetKey("r"))
```

Comment:
we should use the same key/button for picking up and dropping

Reply:
ah maybe 👍 I had it set up a little different when I assigned different keys. Will change that